cumulusci/core/tests/test_config.py

```diff
@@ -1702,6 +1702,247 @@ def test_is_person_accounts_enabled__is_enabled(self):
+        responses.add(
+            "GET",
+            "https://example.com/services/data/v48.0/sobjects/CurrencyType/describe",
+            status=404,
+            json={
+                "errorCode": "NOT_FOUND",
+                "message": "The requested resource does not exist",
+            },
+        )
+
+        # Check 1: is_multiple_currencies_enabled should be False since the CurrencyType describe gives a 404.
+        actual = config.is_multiple_currencies_enabled
+        assert (
+            actual is False
+        ), "config.is_multiple_currencies_enabled should be False since the CurrencyType describe returns a 404."
+        assert (
+            config._multiple_currencies_is_enabled is False
+        ), "config._multiple_currencies_is_enabled should still be False since the CurrencyType describe returns a 404."
+
+        # Check 2: We should still get the CurrencyType describe since we never cached that multiple currencies is enabled.
+        actual = config.is_multiple_currencies_enabled
+        assert (
+            actual is False
+        ), "config.is_multiple_currencies_enabled should be False since the CurrencyType describe returns a 404."
+        assert (
+            config._multiple_currencies_is_enabled is False
+        ), "config._multiple_currencies_is_enabled should still be False since the CurrencyType describe returns a 404."
+
+        # We should have made 3 calls: 1 token call + 2 describe calls
+        assert len(responses.calls) == 1 + 2
+
+    @responses.activate
+    def test_is_multi_currency_enabled__is_enabled(self):
+        config = OrgConfig(
+            {
+                "instance_url": "https://example.com",
+                "access_token": "TOKEN",
+                "id": "OODxxxxxxxxxxxx/user",
+            },
+            "test",
+        )
+
+        assert (
+            config._multiple_currencies_is_enabled is False
+        ), "_multiple_currencies_is_enabled should be initialized as False"
+
+        # Token call.
+        responses.add(
+            "GET", "https://example.com/services/data", json=[{"version": 48.0}]
+        )
+
+        # CurrencyType describe() call.
+        # Since Multiple Currencies is enabled, so the describe call returns a 200.
+        responses.add(
+            "GET",
+            "https://example.com/services/data/v48.0/sobjects/CurrencyType/describe",
+            json={
+                # The actual payload doesn't matter; only matters is we get a 200.
+            },
+        )
+
+        # Add a second describe call to demonstrate we always check the describe until we detect Multiple Currencies is enabled.  From then on, we cache the fact that Multiple Currencies is enabled knowing Multiple Currencies cannot be disabled.
```

Comment:
This is inert since the call isn't being made.

Reply:
@davidmreed , fixed.